packages/babel-core/package.json

```
@@ -54,6 +54,7 @@
     "@babel/types": "workspace:^7.12.10",
     "convert-source-map": "^1.7.0",
     "debug": "^4.1.0",
+    "escape-string-regexp": "condition:BABEL_8_BREAKING ? ^4.0.0",
```

```suggestion
    "escape-string-regexp": "condition:BABEL_8_BREAKING ? ^4.0.0 : ",
```
Seems like the `:` is required. 😄